### PR TITLE
Make misc style updates

### PR DESCRIPTION
**Pull request by arecvlohe**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "graphql-tag": "^2.10.3",
     "mapbox-gl": "^2.0.0",
     "register-service-worker": "^1.6.2",
-    "tailwindcss": "^1.7.3",
+    "tailwindcss": "^1.9.6",
```
Comment (microfauna) [inline package.json:22]: Cool, new tags 😄

File: src/components/Header.vue [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
     <header>
       <nav class="flex py-6 bg-white items-center justify-between">
-        <div class="flex">
+        <div class="flex items-center">
           <img
             src="@/assets/logo.svg" alt="Logo"
             class="flex-shrink-0 h-12 w-12"
           >
-          <h1 class="text-xl sm:text-xl lg:text-4xl text-gray-900 font-bold ml-3">
+          <h1 class="text-xl text-gray-900 font-bold ml-3">
```
Comment (microfauna) [inline src/components/Header.vue]: Did you test this on mobile after removing? (just to be sure)

**Review by microfauna — DISMISSED** [review]
Looks good! We may want to look into some kind of state management down the line for "response". I do a sort of loading-state for when a request is being made in BusinessEditView but it doesn't have to be like that. This is good for now though!

**Review by microfauna — DISMISSED** [review]

File: src/components/Header.vue (hunk 2 above)
Comment (arecvlohe) [reply src/components/Header.vue]: Yeah, it still looks good on mobile, I think the text being that size on all devices is fine

Comment (arecvlohe) [reply src/components/Header.vue]: Actually it will look better w/o it there on mobile

**Pull request merged**